### PR TITLE
Fix: Allow a small delay between initial click focus and subsequent action

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes https://github.com/adaptlearning/adapt_framework/issues/3376

### Fix
* Introduce code to allow a small delay between click and click action at `config.json:_accessibility._options._isClickDelayedAfterFocusEnabled` defaults to `false` to facilitate screen reader focus and dom change processing.

### CODE REVIEW

File: js/.DS_Store
Comment (cahirodoherty-learningpool) [inline js/.DS_Store:1]: .DS_Store committed by accident

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]

File: js/a11y/browserFocus.js [hunk 1]
```diff
@@ -78,8 +79,19 @@ export default class BrowserFocus extends Backbone.Controller {
...
-    $focusable[0].focus();
+    if ($closestFocusable[0] !== document.activeElement) {
+      // Focus on the nearest focusable element if not already with focus
+      $closestFocusable[0].focus();
+    }
+    if (!config._options._isClickDelayedAfterFocusEnabled) return;
+    // Add a small delay to each click to allow screen readers to process focus
+    event.preventDefault();
+    event.stopImmediatePropagation();
+    _.delay(() => {
```
Comment (eleanor-heath) [inline js/a11y/browserFocus.js:92]: would we be better making use of native `setTimeout` here if we are moving away from underscore?

Comment (oliverfoster) [reply js/a11y/browserFocus.js:92]: force of habit, sorry

**Review by eleanor-heath — APPROVED** [review]

**Pull request merged**